app/controllers/candidate_interface/adviser_sign_ups/interruptions_controller.rb

```diff
@@ -0,0 +1,40 @@
+module CandidateInterface
+  module AdviserSignUps
+    class InterruptionsController < CandidateInterfaceController
+      def show
+        @adviser_interruption_form = CandidateInterface::AdviserInterruptionForm.new({ application_form:, proceed_to_request_adviser: params[:proceed_to_request_adviser] })
+        @application_form = application_form
+      end
+
+      def update
+        @adviser_interruption_form = CandidateInterface::AdviserInterruptionForm.new(adviser_interruption_params.merge(application_form:))
+        @application_form = application_form
```

Comment:
This is only necessary if there is an error and we render the show page, so it could be moved into that conditional (line 22) to be more explicit.

Reply:
Or, now that I've read the rest of the code, you could just call `@adviser_interruption_form.application_form.id` where you need it in view since that is the only place it is used, so you don't need this or line 6 at all. But not a blocker, just a bit of noise reduction.